src/components/coupon/CouponPlanDescriptionTabs.tsx

```diff
@@ -83,14 +117,69 @@ const CouponPlanDescriptionTabs: React.FC<{
         </Button>
 
         {mergedCouponCodes.map(couponPlanCode => (
-          <div key={couponPlanCode.id}>
-            <StyledCouponCode className="mr-3">{couponPlanCode.code}</StyledCouponCode>
-            <Typography.Text strong>
-              {`${couponPlanCode.count - couponPlanCode.remaining}/${couponPlanCode.count} ${formatMessage(
-                couponMessages.CouponPlanDescriptionTabs.unit,
-              )}`}
-            </Typography.Text>
-          </div>
+          <StyledCouponCodeBlock key={couponPlanCode.id}>
+            <StyledCouponCode>{couponPlanCode.code}</StyledCouponCode>
+            <StyledCouponCodeAmountBlock>
+              <Typography.Text strong>
+                {`${couponPlanCode.count - couponPlanCode.remaining}/${couponPlanCode.count} ${formatMessage(
+                  couponMessages.CouponPlanDescriptionTabs.unit,
+                )}`}
+              </Typography.Text>
+              <AdminModal
+                renderTrigger={({ setVisible }) => <DeleteOutlined className="ml-4" onClick={() => setVisible(true)} />}
+                title={formatMessage(
+                  couponPlanCode.coupons.some(coupon => coupon.used === true)
```

Comment:
`couponPlanCode.coupons.some(coupon => coupon.used === true)` can be `couponPlanCode.coupons.some(coupon => coupon.used)`

used is boolean